vignettes/writing_vignettes.Rmd

```diff
@@ -181,6 +179,25 @@ restricted either by restricting the source datasets at the beginning of the
 vignette or in the call of `dataset_vignette()` if only some calls are
 affected.
 
+#### Using Footnotes
+
+Footnotes can be useful to add context without adding clutter to the primary subject matter of the vignette being written.
+
+To use footnotes, add a caret and an identifier inside brackets e.g. ([^1]). The identifiers can be numbers or words, but they can’t contain spaces or tabs.
```

Comment:
Should backticks be used for `[^1]`?

It seems that it is rendered differently in admiraldev
![image](https://github.com/pharmaverse/admiraldev/assets/80953585/93d09021-45c9-4936-894f-00f15a9d69b7)

 and admiral
![image](https://github.com/pharmaverse/admiraldev/assets/80953585/719e744c-369c-4379-8a79-51582d91dcae)

Reply:
![image](https://github.com/pharmaverse/admiraldev/assets/43018762/1d73fd62-c6e5-4d0d-ab88-99d1ec49f6f8)

strange... it is showing the same as admiral for me

Reply:
Sorry, the issue was on my side. Some of my packages were out of date. After updating them admiraldev was rendered as admiral.